### PR TITLE
Update model_loader.py

### PR DESCRIPTION
**Pull request by yc-li20**
add
wav2vec 2.0 [base, large];
hubert [base, large];
wavlm [base, baseplus, large];
whisper [base]
to utilize fadtk for speech analysis

### CODE REVIEW

**Comment by yc-li20** [comment]
@microsoft-github-policy-service agree

**Comment by hykilpikonna** [comment]
Thank you so much for adding support for more embedding models! I will review the code changes in a moment.

File: fadtk/model_loader.py [hunk 1]
```diff
@@ -463,6 +463,280 @@ def float32_to_int16(self, x):
...
+            audio = audio[:self.limit]
+
+        inputs = self.processor(audio, sampling_rate=self.sr, return_tensors="pt").to(self.device)
+        with torch.no_grad():
+            out = self.model(**inputs, output_hidden_states=True)
+            out = torch.stack(out.hidden_states).squeeze() # [25 layers, timeframes, 1024]
+            out = out[self.layer] # [timeframes, 1024]
+
+        return out
+
```
Comment (hykilpikonna) [inline fadtk/model_loader.py:535]: I see that the code for W2V2 base and W2V2 large are mostly identical. Would it be better to reuse some duplicate parts of the code by using abstractions?

This also applies to the base and large variants of HuBERT, and WavLM

Comment (yc-li20) [reply fadtk/model_loader.py]: Updated!
I also integrated more model types of _whisper_. Now, the most widely used speech models, i.e.,  _w2v2, hubert, wavlm_, and _whisper_  have been included, which should be sufficient for the majority of speech tasks.

**Review by hykilpikonna — APPROVED** [review]

**Comment by hykilpikonna** [comment]
Thanks! I just merged the pull request. By the way, are there any extra dependencies required for these models? From what I see in the code changes, the only conditional import you used is `transformers`, which is already included in our dependencies, is that correct?

**Comment by yc-li20** [comment]
Yep. No other dependencies required. Thanks for merging!

**Pull request merged**